docs/aspnetcore-mvc.md

```diff
@@ -101,7 +101,32 @@ public IActionResult SignInWithIdp(string idp)
 }
 ```
 
-# Handling failures
+## Accessing OIDC Tokens
+
+To access OIDC tokens, AspNet Core provides the HttpContext.GetTokenAsync(...) extension method.  The following is an example of how to access OIDC tokens from your HomeController:
+
+```csharp
+public class TokenModel
+{
+    public string Name { get; set; }
+
+    public string Value { get; set; }
+}
+
+public class HomeController : Controller
```

Comment:
Do we need to include any specific `using` to use this extension?

Reply:
I'll double check and update as appropriate.